CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Changed
 
+- Immediately invalidate session data on logout
```

Comment:
I don't find this description helpful compared to the commit message. Maybe tell us more about the stakes:
- Avoid lag experienced by user on logout by not waiting for the logout request to succeed